reference-adapter/ifix-department-entity-service/src/main/resources/application.properties

```diff
@@ -6,12 +6,50 @@ org.egov.detailed.tracing.enabled=false
 maximum.supported.department.hierarchy=10
 
 #mongoDB
-spring.data.mongodb.uri=mongodb://localhost:27017/?retryWrites=false
-spring.data.mongodb.database=adapter_master_db
+#spring.data.mongodb.uri=mongodb://localhost:27017/?retryWrites=false
+#spring.data.mongodb.database=adapter_master_db
+
+#Postgres DB
+spring.jpa.hibernate.ddl-auto=update
```

Comment:
As we are not using JPA, remove these configurations

Reply:
ok